crate/utils/src/crypto/symmetric/aes_256_gcm.rs

```diff
@@ -72,10 +65,15 @@ impl EncryptionSystem for AesGcmSystem {
         };
 
         // recover key
-        let key = self.symmetric_key_key_block.key_bytes()?;
+        let key: [u8; Aes256Gcm::KEY_LENGTH] = self
+            .symmetric_key_key_block
+            .key_bytes()?
+            .as_slice()
+            .try_into()?;
```

Comment:
This could be shorten by implementing the correct error conversion:
```suggestion
        let key: [u8; Aes256Gcm::KEY_LENGTH] =
            self.symmetric_key_key_block.key_bytes()?.try_into()?;
```
I think I have seen this pattern before. If you choose to change it, apply it everywhere.

Reply:
done here https://github.com/Cosmian/kms/pull/16/commits/7ebe6f13a6a2589f44430db0be9dd43175ac237b